src/task_2.1/index.ts

```diff
@@ -0,0 +1,111 @@
+import express, { Request, Response, NextFunction } from 'express';
+import uuid from 'uuid/v1';
+import sortBy from 'lodash.sortby';
+import * as Joi from '@hapi/joi';
+import { createValidator } from 'express-joi-validation'
+import UserModel from './models/UserModel';
+import { PORT } from './constants';
+import { userCreateValidationSchema, userUpdateValidationSchema } from './validation';
+
+const users: UserModel[] = [];
+
+const validator = createValidator()
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }))
+
+app.param('id', (req: Request, res: Response, next: NextFunction, value) => {
+    
+    next();
+});
+
+app.get('/user/:id', (req: Request, res: Response, next: NextFunction) => {
+    const { id } = req.params;
+    const user: UserModel = users.find(v => v.id === id && !v.isDeleted);
+
+    if (user) {
+        res.json(user);
+    } else {
+        res.status(400).json(`No user with id ${id} has been found`);
```

Comment:
Resource is absent in the list of resources - 404 error. If not taking it in the context of authorization